components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleOAuth2Authenticator.java

```diff
@@ -215,4 +226,112 @@ public String getFriendlyName() {
     public String getName() {
         return GoogleOAuth2AuthenticationConstant.GOOGLE_CONNECTOR_NAME;
     }
+
+    @Override
+    public String getClaimDialectURI() {
+        String claimDialectUri = GoogleOAuth2AuthenticationConstant.OIDC_CLAIM_DIALECT_URI;
```

Comment:
This is not backward compatible right ? Did this method returned OIDC_CLAIM_DIALECT_URI previously as well ? If the config is not found in the file this will return the value of OIDC_CLAIM_DIALECT_URI

Reply:
Yes. Previously this GoogleOAuth2Authenticator class not override the getClaimDialectUri method in OIDC authenticator. So it executes the implementation in OIDC authenticator. Since I wanted to change the behavior into this authenticator, i have to override it and change it as above. Therefore it is backward compaitible